packages/geometry/src/boundingBox.ts

```diff
@@ -84,3 +84,15 @@ export function union(box: BoundingBox, ...rest: BoundingBox[]): BoundingBox {
   });
 }
 /* eslint-enable padding-line-between-statements */
+
+/**
+ * Returns the distance between the min and max for the provided
+ * bounding box for each axis.
+ */
+export const lengths = (box: BoundingBox): Vector3.Vector3 => {
+  return Vector3.create(
+    Math.abs(box.min.x) + Math.abs(box.max.x),
```

Comment:
Should this be `box.max.x - box.min.x` instead? This would return the wrong value if the min and max are both positive or both negative.

Reply:
Yeah, good point. This was copied over from some other logic, and I'm guessing there's potential for a bug there and we just haven't encountered those edge cases. I'll update this